operator/roles/kiali-deploy/vars/main.yml

```diff
@@ -17,6 +17,13 @@ kiali_vars:
   istio_component_namespaces: "{{ istio_component_namespaces | default(kiali_defaults.istio_component_namespaces) }}"
   istio_namespace: "{{ istio_namespace | default(kiali_defaults.istio_namespace) }}"
 
+  additional_display_details: |
+    {%- if additional_display_details is defined and additional_display_details is iterable -%}
+    {{ additional_display_details }}
```

Comment:
@jmazzitelli I didn't do a dummy copy'n paste here because it was generating an error in kiali operator. Maybe it's due to `combine` function, here `additional_display_details` is an array, unlike the other examples where it's all objects. Might explain the difference? Removing `combine` (hence *not* combining the default and the override) works ; as the default here is an empty array it shouldn't be a problem anyway, however if some day we change the default then it's going to be a problem.